lib/logstash/outputs/dynatrace.rb

```diff
@@ -17,9 +17,9 @@
 require 'logstash/namespace'
 require 'logstash/outputs/base'
 require 'logstash/json'
-require_relative '../../../version'
 
 MAX_RETRIES = 5
+VERSION = '0.2.1'
```

Comment:
```suggestion
PLUGIN_VERSION = '0.2.1'
```
To disambiguate it from the Logstash version.